### ISSUE TITLE
knockout.validation module has stopped working in 3.5.0-rc

### ISSUE THREAD

**Issue by azaslonov** [Knockout-Contrib/Knockout-Validation#663]
Hi guys, knockout validation module has stopped working in 3.5.0-rc, saying this:

```
knockout-latest.js:84 Uncaught Error: ko.applyBindings: first parameter should be your view model; second parameter should be a DOM node
    at a.qc (knockout-latest.js:84)
    at Object.ko.applyBindings (knockout.validation.js:1472)
    at HTMLDocument.<anonymous> (startup.designtime.ts:50)
```
Simple repro: https://jsfiddle.net/azaslonov/4z62mw1r/

Can you guys please have a look?

Just in case, I also posted similar issue into KO repo itself: https://github.com/knockout/knockout/issues/2420

**Comment by nicewk** [comment]
Modify source as below,

var origApplyBindings = ko.applyBindings;
ko.applyBindings = function (viewModel, rootNode) {

	kv.init();
    if (rootNode === undefined) {
        origApplyBindings(viewModel);
    } else {
        origApplyBindings(viewModel, rootNode);
    }
};

**Comment by equisgroup** [comment]
Thanks.

**Comment by crissdev** [comment]
https://github.com/Knockout-Contrib/Knockout-Validation/blob/2.0.3/src/ko.extensions.js#L36

```js
ko.applyBindings = function (viewModel, rootNode) {

	ko.validation.init();

	origApplyBindings(viewModel, rootNode);
};
```

The code should call the original `applyBindings` using the arguments object instead.

```js
ko.applyBindings = function () {
	ko.validation.init();
	origApplyBindings.apply(this, arguments);
};
```

**Comment by cosmoKenney** [comment]
Just came to report this same issue. I can confirm that this is still the case when using the released KO 3.5.0. If I knew how to do a PR, I'd fix. But not sure I can do that or not?? 
For now my work around is to use the second parameter in all my `applyBindings` calls. KO defaults to `window.document.body` (https://github.com/knockout/knockout/blob/aa6447e0f6178ad726c15dcffb79cc9b8fa437be/src/binding/bindingAttributeSyntax.js#L574) when the second parameter is not passed, so calling `ko.applyBindings( myVm, window.document.body );` is the same as calling it without the second parameter.

**Comment by crissdev** [comment]
Fixed by https://github.com/Knockout-Contrib/Knockout-Validation/pull/671 (v2.0.4)
